cmd/podman/containers/exec.go

```diff
@@ -191,6 +193,33 @@ func exec(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// determineTargetCtrAndCmd determines which command exec should run in which container
+func determineTargetCtrAndCmd(args []string, latestSpecified bool, execCidFileProvided bool) (string, []string, error) {
+	var nameOrID string
+	var command []string
+
+	if len(args) == 0 && !latestSpecified && !execCidFileProvided {
+		return "", nil, errors.New("exec requires the name or ID of a container or the --latest or --cidfile flag")
+	} else if latestSpecified && execCidFileProvided {
+		return "", nil, errors.New("--latest and --cidfile can not be used together")
+	}
+	command = args
+	if !latestSpecified {
+		if !execCidFileProvided {
+			// assume first arg to be name or ID
+			command = args[1:]
+			nameOrID = strings.TrimPrefix(args[0], "/")
+		} else {
+			content, err := os.ReadFile(execCidFile)
```

Comment:
Did I miss checks to make sure the file provided is secure?  I would expect an absPath kind of check at least.

Reply:
@TomSweeneyRedHat thanks for the feedback. I'd agree with @Luap99 on this one.